### PR TITLE
implemented smooth scrolling

### PR DESCRIPTION
**Pull request by sp9324**
Implemented smooth scrolling. When "Games" button is clicked, library react-scroll creates a smooth scroll effect to the games section. Also changed CSS to match the effect.

https://github.com/user-attachments/assets/dc23c526-3efe-4f7c-8c4f-8c28d905d2ba

### CODE REVIEW

File: src/components/Home/CardItem.js [hunk 1]
```diff
@@ -24,9 +24,14 @@ const CardItem = (props) => {
...
-          <Link to={props.path} class="btn btn-primary">
+          <ScrollLink
+            to={props.path}  
+            smooth={true}
+            duration={500}
+            className="btn btn-primary"
+          >
             Play
-          </Link>
+          </ScrollLink>
```
Comment (ken1000minus7) [inline src/components/Home/CardItem.js:34]: This is supposed to take the user to the page of the particular game/activity so scroll link is not required here

File: src/pages/Games.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
         <div className="games-root">
             <h1 className="games-title">Games</h1>
             <div className="games-content">
                 {
-                    games.map(game => {
+                    games.map((game, index) => {
                         return (
-                            <GameCard game={game} />
-                        )
+                            <section id={`game-${index}`} key={game.urlTerm} style={{ padding: '50px 0' }}>
```
Comment (ken1000minus7) [inline src/pages/Games.js:13]: Can u remove this padding, it doesn't [look good](https://acm-fun-git-fork-sp9324-master-acm-fun.vercel.app/games)

File: src/pages/Home.js [hunk 3]
```diff
@@ -4,8 +4,11 @@ import Cards from "./home/Cards"
 export const Home = () => {
     return (
         <>
-            <HeroSection/>
-            <Cards/>
+            <HeroSection />
+            <section id="games-section" style={{ padding: '100px 0' }}>
+                <h2>Games Section</h2>
```
Comment (ken1000minus7) [inline src/pages/Home.js:9]: The cards below show both games and activities so naming this just games would be incorrect
Also the text is kinda invisible cuz of the background
image

File: src/components/Home/HeroSection.js [hunk 4]
```diff
@@ -1,20 +1,31 @@
...
+          Games
+        </ScrollLink>
+        <ScrollLink
+          to="activities-section" 
+          smooth={true}
+          duration={500}
+          className="btn btn-light"
+        >
+          Activities
+        </ScrollLink>
```
Comment (ken1000minus7) [inline src/components/Home/HeroSection.js:27]: This button is currently doing nothing cuz games and activities are shown together below
You can either direct it to `games-section` as well or separate the cards below into games and activities and link accordingly

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]

**Comment by sp9324** [comment]
Closing this issue as the change made is unnecessary.

**Pull request closed without merging**